scanner/src/lib/dataproducts/LogisticsEmissions_v0_1.svelte

```diff
@@ -165,36 +168,19 @@
       />
     {/each}
   {/if}
-</article>
+</Article>
 
 <style lang="scss">
-  article {
-    color: white;
-    font-style: normal;
-
-    .title {
-      font-size: 1rem;
-      font-weight: 500;
-      line-height: 1.5rem;
-      margin-bottom: 1rem;
-    }
-
-    .leg-id-label {
-      font-size: 0.75rem;
-      line-height: 1.5rem;
-      color: #798893;
-      text-transform: uppercase;
-    }
-
-    .leg-id {
-      font-size: 1rem;
-      line-height: 1.5rem;
-      margin-bottom: 1rem;
-    }
-
-    .divider {
-      border-bottom: 1px solid #20303e;
-      margin-bottom: 1rem;
-    }
+  .leg-id-label {
+    font-size: 0.75rem;
+    line-height: 1.5rem;
+    color: #798893;
```

Comment:
This color should likely be an scss variable 
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/ee5f38a3-89e7-45f4-a084-b6222c8b714d)